lib/buildkite/builder/dsl/features/trigger.rb

```diff
@@ -4,7 +4,7 @@ module Dsl
       module Features
         module Trigger
           def trigger(template = nil, **args, &block)
-            add_to_steps(Pipelines::Steps::Trigger, template, **args, &block)
+            Helpers.add_to_steps(_context, Pipelines::Steps::Trigger, template, **args, &block)
```

Comment:
Now that you have access to the context, should there just be a public api on pipeline or group to add to steps?